azurerm/internal/services/netapp/data_source_netapp_volume.go

```diff
@@ -61,6 +62,18 @@ func dataSourceArmNetAppVolume() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+
+			"protocol_types": {
```

Comment:
if only one is allowed this should be
```suggestion
			"protocol": {
```

I think type isn't really needed either ?

Reply:
This name aligns with all other client tools, e.g. SDKs, ARM Templates, Portal, Powershell and CLI so this name will be very familiar to the users.

Reply:
About the documentation, I'll look at it as well.